api/user/author/authors.py

```diff
@@ -157,6 +159,37 @@ def register_user():
         # username already exists
         return {"message": "User Already exists"}, 409
 
+    user = Author(
+        username=username,
+        password=bcrypt.generate_password_hash(password).decode("utf-8"),
+        host=request.host,
+        approval=Approval.PENDING,
```

Comment:
shouldn't default to approved? or pull a config value here?

Reply:
yeah config is the way to go, maybe you can add that?